pkg/overlay/mocks/overlay.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2018 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package mock
+package mock_overlay
```

Comment:
Usually it's nicer to avoid underscores in package names. https://golang.org/doc/effective_go.html#package-names

Reply:
https://blog.golang.org/package-names